src/app/body/milestone-details/milestone-details.component.html

```diff
@@ -153,6 +153,38 @@
             </div>
           </div>
         </div>
+        <div class="process-wrapper" *ngIf="milestoneService.taskData.length > 0 && milestoneService.incompleteTasks == 0">
+          <div id="progress-bar-container">
+            <ul>
+              <ng-container *ngFor="let item of milestoneService.completedTasks">
+                <a (click)="openTaskDetails(item.Id)">
```

Comment:
Instead of creating a new progress bar for completed graphs.
Try to implement a single progress bar.